include/rangers-api/ApplicationCommon/FSM/GOCHsm2.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace app_cmn::fsm {
+    class GOCHsm2 : public hh::game::GOComponent {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.